### PR TITLE
{bio}[GCCcore/9.3.0] T-COFFEE v13.45.0

### PR DESCRIPTION
**Pull request by kErica**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/t/T-COFFEE/T-COFFEE-13.45.0-GCCcore-9.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
```
Comment (boegel) [inline easybuild/easyconfigs/t/T-COFFEE/T-COFFEE-13.45.0-GCCcore-9.3.0.eb:1]: @kErica Please fix this URL:

```suggestion
# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/t/T-COFFEE/T-COFFEE-13.45.0-GCCcore-9.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+modextravars = {
+    'EMAIL_4_TCOFFEE': 'dummy@email.com',
+    'MAX_N_PID_4_TCOFFEE': '9999999',
+}
+
+modluafooter = """
+setenv("DIR_4_TCOFFEE", os.getenv('PWD'))
+setenv("CACHE_4_TCOFFEE", os.getenv('PWD') .. "/.t_coffee/cache/")
+setenv("TMP_4_TCOFFEE", os.getenv('PWD') .. "/.t_coffee/tmp/")
+setenv("LOCKDIR_4_TCOFFEE", os.getenv('PWD') .. "/.t_coffee/lck/")
```
Comment (boegel) [inline easybuild/easyconfigs/t/T-COFFEE/T-COFFEE-13.45.0-GCCcore-9.3.0.eb:55]: @kErica I'm not sure it's a good idea to define these `*_TCOFFEE` environment variables based on the current directory at the time of loaded the module...

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/t/T-COFFEE/T-COFFEE-13.45.0-GCCcore-9.3.0.eb (hunk 2 above)
Comment (kErica) [reply easybuild/easyconfigs/t/T-COFFEE/T-COFFEE-13.45.0-GCCcore-9.3.0.eb]: @boegel I have changed it to $HOME

**Review by boegel — CHANGES_REQUESTED** [review]
@kErica When using `GCCcore` as toolchain, you must also include `binutils` as a build dependency

**Comment by lexming** [comment]
Closing as this PR targets a very old toolchain and there are already easyconfigs for newer versions of [T-Coffee](https://github.com/easybuilders/easybuild-easyconfigs/tree/develop/easybuild/easyconfigs/t/T-Coffee)

**Pull request closed without merging**